apps/judicial-system/backend/src/app/modules/indictment-count/test/delete.spec.ts

```diff
@@ -2,7 +2,7 @@ import { uuid } from 'uuidv4'
 
 import { createTestingIndictmentCountModule } from './createTestingIndictmentCountModule'
 
-import { DeleteIndictmentCountResponse } from '../models/delete.response'
+import { DeleteIndictmentCountResponse } from '../models/deleteIndictmentCount.response'
```

Comment:
Please consider sharing a single delete response as discussed in the api project.